---
layout: default
title: "Research"
permalink: /research/
---

<div class="container-sm">
    <div class="row">
        <div class="col-sm-8 offset-2">
            <div class="row g-0">
                <div class="col-10 py-4">
                    <h5>Research Summary</h5>

                    <p>Our laboratory is looking at how cells ensure that proteins fold into their correct shape, as well as the role of protein misfolding in disease and normal physiology. We are also developing experimental and analytical approaches for exploring the organizational principles of biological systems and globally monitoring protein translation through ribosome profiling.</p>

                    <p>To ensure proper folding, cells have evolved a sophisticated and essential machinery of proteins called molecular chaperones that assist the folding of newly made polypeptides. The importance of proper protein folding is underscored by the fact that a number of diseases, including Alzheimer's and those involving infectious proteins (prions), result from protein-misfolding events. My research focuses on identifying and understanding the machinery necessary for efficient folding, as well as studying the mechanism and consequences of protein misfolding. We are also developing experimental and analytical approaches for exploring the organizational principles of complex biological systems and for monitoring protein translation in vivo with unprecedented precision and depth.</p>

                    <p>The ability to bridge large-scale approaches and mechanistic investigations is a key focus of my lab. Beyond the immediate payoff of these efforts, I hope our work will contribute to the broader goal of developing more principled, less ad hoc approaches for defining gene functions.</p>

                    <p>Below are descriptions of three broad project areas of the lab.</p>
                </div>
                <div class="col-2">
                    <img class="research_img" src="{{ site.baseurl }}/assets/img/old_site_images/research/proteomics.jpg">
                </div>

            </div>
            <div class="row border_top g-0">
                <div class="col-2">
                    <img class="research_img" src="{{ site.baseurl }}/assets/img/old_site_images/research/library1.jpg" style="max-height:100%; max-width:100%; overflow: hidden;">
                </div>
                <div class="col-10 py-4">
                    <h5>Ribosome Profiling</h5>

                    <p>The advent of microarray technology, and more recently RNA-seq, has made it possible to monitor the internal state of a cell with unprecedented precision. However, these approaches monitor messenger RNA (mRNA) levels, whereas for the large majority of cases it is proteins that are directly responsible for mediating a gene's cellular functions. We know that mRNA levels are often a highly imperfect proxy for protein production due to the extensive use of translational control. Our understanding of the strategies used to regulate translation lags far behind that of transcription, largely due to the difficulty in monitoring protein synthesis rates.</p>

                    <p>We have developed a ribosome profiling approach based on deep-sequencing of ribosome-protected fragments that makes it possible to determine the rate of translation with a depth, speed and accuracy that rivals or exceeds existing approaches for following mRNA levels. Ribosome profiling also provides a powerful tool for dissecting the molecular mechanism of translation in vivo.</p>

                    <p>We have now developed a suite of techniques based on ribosome profiling and have applied these techniques to address a number of fundamental questions including: (1) Development of ribosome profiling protocols for a wide variety of eukaryotic and prokaryotic organisms. (2) Uses of ribosome profiling to globally monitor when chaperones, targeting factors or processing enzymes engage nascent chains. (3) Development of a strategy for monitoring subcellular translation. (4) Deciphering the driving force and biological consequences underlying the choice of synonymous codons. (5) Defining the protein coding potential of complex genomes.</p>
                </div>

            </div>
            <div class="row border_top g-0">
                <div class="col-10 py-4">
                    <h5>Protein Folding in the Endoplasmic Reticulum</h5>

                    <p>As a rule, proteins that enter the secretory pathway fold within the ER. The ER establishes and maintains a highly specialized environment optimized for folding. Understanding how this is accomplished is a major focus of our research.</p>

                    <p>But of course the ER is much more than just the entry point to the secretory pathway. This organelle integrates lipid biosynthesis, protein folding, glycosylation, quality control and degradation and plays a key role in cellular homeostasis, infection, and immune defense. Accordingly, our efforts have taken a more holistic approach, encompassing all aspects of ER function.</p>

                    <p>Major recent findings include the following:</p>

                    <p><b><i>▶ Identification and characterization of Yos9 as a sugar sensor of misfolded proteins</i></b></p>

                    <!-- How misfolded forms are recognized and distinguished from the myriad of folding substrates represents a fascinating problem in discrimination. Cells must maintain a balance between overly promiscuous destruction of slow-folding proteins, while preventing escape of toxic forms. For most luminal proteins, this discrimination depends not only on the folding status of a polypeptide, but also on its glycosylation state.

                    We, and others concurrently, identified the conserved luminal protein Yos9 as the lectin sensor responsible for probing the glycosylation state of misfolded proteins. This allowed us, in a series of papers, to work out the elaborate mechanism used to correctly mark and degrade terminally misfolded proteins. This strategy of marking potential substrates by a sugar modification and decoding the signal in a second recognition event provides a proofreading mechanism for enhancing substrate specificity. We are now recapitulating this modification and recognition cascade in vitro from purified components. I am particularly excited about this work as it represents a terrific opportunity to provide our most in-depth structural and mechanistic understanding to the fascinating question of how the ER tells right from wrong.-->

                    <p><b><i>▶ Discovery of a novel branch of the metazoan UPR involving targeted mRNA destruction</i></b></p>

                    <!-- Elegant work pioneered by my UCSF/HHMI colleague and collaborator Peter Walter identified Ire1 as a key ER sensor of misfolded proteins. These studies revealed the canonical activation pathway in which unfolded proteins sensed in the ER lumen induce the cytoplasmic nuclease domain of Ire1 to cleave the mRNA encoding XBP-1 (Hac1 in yeast), enabling splicing and production of active transcription factor. This allows for the remodeling of the ER in response to stress. But transcriptional responses are slow. We found that in metazoans, activation of the Ire1 endonuclease independently induces the rapid turnover of many mRNAs encoding membrane and secreted proteins through a pathway we call regulated Ire1-dependent decay (RIDD). This response is well suited to complement other UPR mechanisms by selectively halting production of proteins that challenge the ER and clearing the translocation and folding machinery for the subsequent remodeling process. More recently, we, and independently the Papa group, provided evidence that cells use a multi-tiered mechanism by which different conditions in the ER lead to distinct outputs (transcriptional versus RIDD) from Ire1. This allows for a nuanced response to different stresses and provides the potential for therapeutics that "surgically" intervene with specific Ire1 outputs, preventing proapoptotic effects while keeping protective outputs intact. -->

                    <p><b><i>▶ Identification of the GET pathway: a conserved system responsible for the biogenesis of tail-anchored membrane proteins</i></b></p>

                    <!-- Tail-anchored (TA) proteins, defined by the presence of a single C-terminal transmembrane domain (TMD), play critical roles throughout the secretory pathway and in mitochondria. Because of its position near the C terminus, the TMD of TA proteins is occluded by the ribosome until translation is completed. Thus, TA proteins cannot exploit the classic cotranslational SRP/Sec61 translocation mechanism used by most secretory pathway proteins. Starting from our Genetic Interaction Map approach (see below), we identified the Guided Entry of Tail-anchor (GET) proteins as three components of a conserved machinery responsible for the biogenesis of TA proteins. Further genetic interactions maps identified two additional ribosome-associated components (Get4 and Get5). These and other observations suggest a pathway for the biogenesis of TA proteins in which Get4/5 recognize TA proteins as synthesis is completed. The TA is then delivered to a soluble ATPase, Get3, which is subsequently recruited to the ER membrane by a receptor composed of the Get1/2 proteins, thereby allowing proper membrane insertion. -->

                    <p><b><i>▶ Discovery of a molecular caliper mechanism for determining the length of very long-chain fatty acids</i></b></p>

                    <!-- Despite their numerous essential functions, our understanding of how very long-chain fatty acids (VLCFAs) are made was highly limited. From an academic standpoint, the chain length diversity of VLCFAs, which enables their functional diversity, represents a fascinating example of how polymerization of simple building blocks can be used to build chemically complex macromolecules. From a practical standpoint, there are potential therapeutic and commercial benefits to be gained from manipulating VLCFA synthesis. A key obstacle to these endeavors has been the fact that VLCFAs are made by a collection of detergent-labile, multi-enzyme complexes embedded in the ER membrane. Consequently, even though the basic outline of the chemistry underlying VLCFA synthesis was established in the 1960â€™s, the minimal set of proteins, including the key dehydratase component, needed for VLCFA synthesis was not known.

                    We identified Yjl097w (Phs1p) as the founding member of a new family of membrane proteins that act as the VLCFA dehydratase. This missing piece allowed us to reconstitute VLCFA synthesis in proteoliposomes containing purified Phs1p and three other membrane proteins. We used this in vitro system to address what is arguably the most important question regarding VLCFA synthesis: how does the biosynthetic machinery instruct the precise number of two-carbon additions that yields a defined length product. Our studies revealed a fascinating mechanism, akin to a caliper, in which the chain length is determined by the distance between the cytosolic active site that mediates two-carbon addition and a lysine near the luminal end of a transmembrane helix. By stepping this lysine residue along one face of the helix toward the cytosol, we engineered novel synthases with correspondingly shorter VLCFA outputs. -->

                    <p><b><i>▶ Identification of the Orm family of proteins as critical mediators of sphingolipid homeostasis</i></b></p>

                    <!-- Although recent studies have shed light on the critical roles of sphingolipids both as structural components of membranes and critical signaling molecules, how cells sense and regulate the levels of sphingolipids has remained a mystery. We found that members of the highly conserved but previously uncharacterized Orm protein family inhibit the first and rate-limiting step of de novo sphingolipid production. Moreover, the activity of Orm proteins is dynamically regulated by a key feedback loop: when sphingolipid synthesis is disrupted, Orm proteins are inactivated by phosphorylation, thus providing cells with an elegant mechanism to couple sphingolipid production to metabolic demand.

                    Our findings are of particular interest in light of the recent identification of a major genetic risk factor for asthma near the locus of the human ORM gene homolog ORMDL3. Polymorphisms which alter ORMDL3 expression account for up to ~20% of childhood asthma cases and have also been implicated in other inflammation-based diseases such as Type I diabetes. Thus, our finding that alterations in ORM gene activity profoundly affect sphingolipid metabolism raises the testable hypothesis that misregulation of sphingolipids contributes directly to the pathogenesis of asthma. We are now extending our initial studies to define the mechanism by which Ormdl proteins are regulated in mammalian cells. Recent progress in identifying upstream components of the yeast phosphorylation feedback pathway will guide these efforts and may also enable us to define a molecular sensor of sphingolipid levels. Thus, our work on Orm family proteins promises to inform a fundamental question in cell biology and may also reveal novel roles for sphingolipids in inflammatory diseases. -->


                </div>
                <div class="col-2">
                    <img class="research_img" src="{{ site.baseurl }}/assets/img/old_site_images/research/ER2.jpg">
                </div>

            </div>
            <div class="row border_top g-0">
                <div class="col-2">
                    <img class="research_img" src="{{ site.baseurl }}/assets/img/old_site_images/research/emap1.jpg">
                </div>
                <div class="col-10 py-4">
                    <h5>Understanding the Organizational Principles of Cellular Systems</h5>

                    <p><b><i>Construction and analysis of high-density genetic interaction maps</i></b> A major challenge of the post genomic era is to understand how genes function together in normal and disease states. In addition to a description of the molecular activity of individual proteins, a comprehensive description of a cell requires an understanding of how groups of proteins act together to carry out specific biological processes, and how those different processes communicate among themselves. Theoretical considerations suggest that information regarding each of these organizational levels can be obtained from the analysis of genetic interaction maps (pairwise descriptions of the extent to which the loss of one gene will aggravate or buffer the effect of the loss of a second one). The spectrum of genetic interactions is also crucial for guiding efforts to tailor drug treatments to an individual's genetic makeup, and for developing rational strategies for drug-combination therapies.</p>

                    <p>We have used an integrated set of experimental and computational strategies that now make it possible to measure and analyze high-density genetic interaction maps in a variety of microorganisms including S. cerevisiae, S. Pombe, and E. coli. The application of this strategy to several fundamental processes, including the yeast early secretory pathway, has fueled a string of discoveries opening up new research directions and providing molecular insights into long-standing questions. including our advances listed above in the understanding of the ER.</p>

                    <p><b><i>CRISPRi: A general tool for programmed control of gene expression</i></b> Together with my UCSF and UC Berkeley colleagues Stanley Qi, Wendell Lim, Jennifer Doudna, and Adam Arkin, we have developed the CRISPR-associated catalytically inactive dCas9 protein as a general platform for RNA-guided DNA targeting. We have shown that fusion of dCas9 to effector domains with distinct regulatory functions enables stable and efficient transcriptional repression or activation in human and yeast cells with the site of delivery determined solely by a co-expressed short guide (sg)RNA. Coupling of dCas9 to a transcriptional repressor domain can robustly silence expression of multiple endogenous genes. Our results established that the CRISPR system can be used as a modular and flexible DNA-binding platform for the recruitment of proteins to a target DNA sequence and reveal the potential of CRISPRi as a general tool for the precise regulation of gene expression in eukaryotic cells.</p>

                </div>
            </div>
            <div class="row border_top g-0">
                <div class="col-10 py-4">
                    <p><b><i>Mammalian Genetic Interaction Maps</i></b> A convergence of technologies now makes it possible to construct similar genetic interaction maps in mammalian cells. These technologies include: (1) Efficient lentiviral vectors allowing stable knock-down of one or more genes either using shRNAs or, more recently, CRISPR technology, (2) Highly parallel microarray-based oligosynthesis, enabling production of massive lentiviral RNA libraries, (3) Deep sequencing technology that makes it possible to monitor simultaneously the phenotypic consequences of each of the individual elements in such highly complex libraries. We have developed an integrated strategy for generating and quantitatively monitoring complex pooled lentiviral libraries. We have constructed an ultracomplex (25 shRNAs/gene and 500,000 total shRNAs) whole genome shRNA library. These ultracomplex libraries let us overcome the two common problems that have plagued shRNA screens: high false-negative rates (due to low efficacy of many shRNAs) and high false-positive rates (due to off-target effects). Our high-coverage library maximizes the chances of effectively targeting each gene with multiple independent shRNAs, and thus allows rigorous evaluation of the probability that shRNAs act through the intended target gene.</p>

                    <p>Using these ultracomplex library, we have developed an integrated methodology for genome-wide identification of genes with relevant phenotypes and systematic mapping of all genetic interactions (GIs) between them. We have demonstrated the potential of this approach in an application to pathways controlling the susceptibility of human cells to the toxin ricin. We have also developed a quantitative framework for our strategy, including experimental design, derivation of quantitative phenotypes from pooled screens, robust identification of hit genes using ultra-complex shRNA libraries, parallel measurement of hundreds of thousands of GIs from a single double-shRNA experiment and construction of GI maps. Our pooled approach enables rapid screening of the same shRNA library in different cell lines and under different conditions to determine a range of different phenotypes.</p>

                    <p>We are particular interested in using the above approach to map out how chaperones and stress networks contribute to the ability of transformed cancer cells to thrive despite being in highly stressed states.</p>
                </div>
                <div class="col-2">
                    <img class="research_img" src="{{ site.baseurl }}/assets/img/old_site_images/research/psi.jpg">
                </div>
            </div>
        </div>
    </div>
</div>