---
layout: default
title: "Resources"
permalink: /resources/
---


<div class="container">
    <div class="col-md-8 offset-2">

        <div class="row border_top py-2 mt-4">
            <h5>Contact Information</h5>

            <p>455 Main St<br>
            Whitehead Institute Suite 661B<br>
            Cambridge, MA 02142</p>

            <p><a href="https://wi.mit.edu/">Whitehead Institute</a></p>
            <p><a href="https://biology.mit.edu/">MIT Biology</a></p>
            <p><a href="https://www.hhmi.org/">Howard Hughes Medical Institute</a></p>


            <p>Addgene: <a href="https://www.addgene.org/search/catalog/plasmids/?q=Weissman+Jonathan">Plasmids</a></p>
            <p>Addgene: <a href="https://www.addgene.org/search/catalog/pooled-libraries/?q=Weissman+Jonathan">Pooled Libraries</a></p>

        </div>

        <div class="row border_top py-2 mt-4">
            <h5>New Resources</h5>

            <div class="col">
                <h5><a href="https://gwps.wi.mit.edu/">Genome-Wide Perturb-Seq</a></h5>
                <p>Perturb-seq is an experimental approach to map the transcriptional effects of genetic perturbations by combining
                CRISPR-based genetic screening with information-rich, single-cell RNA-sequencing phenotypes.</p>

                <p>Applied at genome-scale, Perturb-seq enables systematic prediction of gene function and in-depth study of complex
                cellular phenotypes such as aneuploidy, differentiation, RNA processing, and stress-specific regulation of the mitochondrial genome.</p>
            </div>
            <div class="col">
                <img src="/assets/img/gwps.png" style="width: 400px;">
            </div>


        </div>


        <div class="row border_top py-2 mt-4">
            <h5>Software from the Weissman Lab</h5>

            <p><a href="http://gimap.ucsf.edu/"><b><i>GImap</i></b> software for analysis of pooled screens and genetic interaction maps in mammalian cells</a></p>

            <p><a href="https://github.com/alexfields/ORF-RATER/"><b><i>ORF-RATER</i></b> package for coding sequence annotation based on ribosome profiling data</a></p>

            <p><a href="https://github.com/joshuagryphon/plastid/"><b><i>Plastid</i></b> python library for analysis of high-throughput sequencing and other genomic data</a></p>

            <p><a href="https://github.com/mhorlbeck/ScreenProcessing"><b><i>ScreenProcessing pipeline</i></b> for analysis of pooled genetic screens</a></p>

        </div>

        <div class="row border_top py-2 mt-4">
            <h5>Other links</h5>

            <p><a href="/crispr/">CRISPRi/a screening materials</a></p>

            <p><a href="https://yeastgfp.yeastgenome.org/">Yeast GFP-fusion localization database</a></p>

            <p><a href="https://www.yeastgenome.org/">Saccharomyces Genome Database</a></p>

            <p><a href="https://www.yeastgenome.org/">Saccharomyces Genome Database</a></p>

        </div>
    </div>
</div>